target/linux/bcm27xx/bcm2710/config-5.10

```diff
@@ -51,6 +49,7 @@ CONFIG_ARM_GIC_V3_ITS_PCI=y
 CONFIG_ARM_PSCI_FW=y
 CONFIG_ARM_RASPBERRYPI_CPUFREQ=y
 CONFIG_ARM_TIMER_SP804=y
+# CONFIG_ASN1 is not set
```

Comment:
Please move this to target/linux/generic/config-5.10
Since 63f32eb88ee6a4ed96bf28e9b93383e8064a0597 this option is available.

Reply:
`# CONFIG_OID_REGISTRY is not set` is already set in target/linux/generic/config-5.10